security/encryption.md

```diff
@@ -16,5 +16,5 @@ Check the quality of the Tradecloud certificate using [SSL Labs](https://www.ssl
 
 ## Encrypted and hashed tokens
 
-The Tradecloud API uses hashed [JSON Web Tokens](https://jwt.io/) therefor the token cannot be changed.
+The Tradecloud API uses hashed [JSON Web Tokens](https://jwt.io/) therefore the token can only be compromised.
```

Comment:
What do you mean with "can only be compromised." (does not sound good)

Reply:
I rephrased this sentence too many times; and ended up with a mess. Should be better now.